sky/engine/BUILD.gn

```diff
@@ -22,6 +22,13 @@ config("config") {
   cflags = [
     "-Wno-inconsistent-missing-override",
   ]
+
+  if (is_fuchsia) {
+    # TODO(abarth): Remove once the kernel support unmapping memory properly.
+    defines = [
+      "MEMORY_TOOL_REPLACES_ALLOCATOR",
```

Comment:
This reminds me to remove the flag on iOS as well and get rid of the `mprotect`s to see if I can get any measurable performance difference using the different allocators.